src/CallManager.cpp

```diff
@@ -392,6 +400,21 @@ CallManager::callsSupported() const
 #endif
 }
 
+bool
+CallManager::screenShareSupported()
+{
+        return std::getenv("DISPLAY") != nullptr;
```

Comment:
I think you also need to check that WAYLAND_DISPLAY is not set, to make sure we are not running under XWayland?

Reply:
Ah ok, will do.